wuvt/admin/views.py

```diff
@@ -492,6 +492,29 @@ def donation_index():
         if 'reset_stats' in request.form:
             redis_conn.set('donation_stats_start',
                            datetime.datetime.utcnow().isoformat())
+        if 'set_paid' in request.form:
+            # TODO log who set as paid
+            o = Order.query.get(request.form["id"])
+            o.set_paid(method="check")
+            db.session.add(o)
```

Comment:
What is being added here? This call doesn't seem correct; removing it should work.